src/tasks/platforms/crc.ts

```diff
@@ -54,6 +57,16 @@ export class CRCHelper {
       },
       VersionHelper.getOpenShiftCheckVersionTask(flags),
       VersionHelper.getK8sCheckVersionTask(flags),
+      {
+        title: 'Verify amount "crc" users',
```

Comment:
Do we really care about users number?
Would it better to just `Verify the existence of users?`